myriad-scheduler/src/main/java/org/apache/myriad/MyriadModule.java

```diff
@@ -168,4 +172,16 @@ ExecutorCommandLineGenerator providesCLIGenerator(MyriadConfiguration cfg) {
     }
     return cliGenerator;
   }
+  
+  protected MyriadConfiguration generateMyriadConfiguration(String configFile) {
+    ObjectMapper mapper = new ObjectMapper(new YAMLFactory());
```

Comment:
I think a method like this exists in TaskUtils (which may not be the right place for it).  Does it meet your needs?